db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_tenants.rb

```diff
@@ -3,7 +3,7 @@
 #
 tenant_list = {nil => "default"}
 service_template = $evm.root.attributes["service_template"]
-if service_template.orchestration_manager
+if service_template.respond_to?(:orchestration_manager)
```

Comment:
@jrafanie A good catch. It is weird sometimes a simple fix requires multiple rounds of work :(
Because the dialog could be (incorrectly) used anywhere, we want to prevent any possible user errors. Here we want to make sure every object is not `nil` before we can call into a method.